kolibri/core/deviceadmin/utils.py

```diff
@@ -41,6 +41,19 @@ def default_backup_folder():
     return os.path.join(KOLIBRI_HOME, "backups")
 
 
+def get_backup_files(kolibri_version):
```

Comment:
It's possible that we have backups from many previous versions, and most backups will be from previous versions. So when we are cleaning up backups, filtering to one specific Kolibri version is unlikely to save us much space.

Instead we probably want to get all the backups that are _not_ from this version, as that's most likely the backup we just created.